### PR TITLE
chore: added nvmrc

### LINKED ISSUE #575: Including a `.nvmrc` file for the project
#### Questions

- Why we need this improvement?
Since, this project specifies the node version to be >= 14

- How will this change help?
 Some developers like me use different versions of node for different projects, hence having a `.nvmrc` file would automatically be helpful in preventing human errors :smile: .  If this file exists, we just have to load the specified version using it. 

#### Description
- What changes have to be introduced?
We just need to add a `.nvmrc` file in the root directory.

- Will this be a breaking change?
 No, It's just a specification file.

- How could it be implemented/designed?
 By running the command : `node -v > .nvmrc `

### PR DESCRIPTION
**Pull request by ritik307**
**Description**
- Added command to implement and install `.nvmrc`.
- Updated `docs/development.md` with the commands to setup `.nvmrc`  

**Related issue(s)**
Resolves #575

### CODE REVIEW

**Comment by ritik307** [comment]
@fmvilas @derberg @ron-debajyoti  kindly review the PR

**Comment by ron-debajyoti** [comment]
> @fmvilas @derberg @ron-debajyoti kindly review the PR

I'm not a maintainer tbh, just another fellow contributor :smile:   @jonaslagoni would the main guy here.

**Review by jonaslagoni — COMMENTED** [review]
@ritik307 is there any reason you see us needing the script `nvmrc` rather than just committing that file? That way the steps others need to do it only `nvm use` 🤔

**Comment by ritik307** [comment]
> @ritik307 Thanks for taking the time to do this +1
> 
> One question, is there any reason you see us needing the script `nvmrc` rather than just committing that file? That way the steps others need to do it only `nvm use` thinking

I think it better bcz there won't be a need for a dev to have nvm already installed in their machine and also if a dev doesn't require nvmrc file they still have to keep it in the root folder regardless. I don't have any problem switching to the way you are thinking to implement this I will update the PR accordingly.

**Comment by jonaslagoni** [comment]
> I think the implemented approach is better bcz there won't be a need for a dev to have nvm already installed in their machine and also if a dev doesn't require nvmrc file they still have to keep it in the root folder regardless

Regardless of this approach or the other, would they not need `nvm` installed anyway? 

The only difference I see is that, with your script, we can "dynamically` create the appropriate nvmrc file. However, as the node requirement wont really change, having it hardcoded might be a better fit 🙂

**Comment by ritik307** [comment]
> > I think the implemented approach is better bcz there won't be a need for a dev to have nvm already installed in their machine and also if a dev doesn't require nvmrc file they still have to keep it in the root folder regardless
> 
> Regardless of this approach or the other, would they not need `nvm` installed anyway?
> 
> The only difference I see is that, with your script, we can "dynamically` create the appropriate nvmrc file. However, as the node requirement wont really change, having it hardcoded might be a better fit slightly_smiling_face

Done! kindly review

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Got a few follow up questions 🙂 

Also for the title of the PR, please use `chore:` as this is not a feature for the library, but a feature for the development setup 🙂

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v14.18.2
```
[the remark this answers is not in the document]
Comment (ritik307) [reply .nvmrc:1]: I am not sure about the hardcode common practice but I am sure that v14 would be enough and supported

[the remark this answers is not in the document]
Comment (jonaslagoni) [reply .nvmrc:1]: Let's try with only hardcoding the major version, for now, that way we take full advantage of node 🙂 
```suggestion
v14
```

[the remark this answers is not in the document]
Comment (ritik307) [reply .nvmrc]: done!

File: docs/development.md [hunk 2]
```diff
@@ -12,10 +14,13 @@ You can either build the image and run the needed commands manually or rather us
 ## Environment setup
 
 To setup the environment follow these steps:
+
 1. Setup the project by first installing the dependencies `npm install`
-2. Make sure the tests pass by running `npm run test` script
-    - You can update snapshots by running `npm run test -- -u`
-3. Make sure code is well formatted and secure `npm run lint`
+2. Run `nvm use` to update the nodejs version.
```
[the remark this answers is not in the document]
Comment (ritik307) [reply docs/development.md]: Done!

**Review by jonaslagoni — APPROVED** [review]
LGTM! 🎉

**Comment by jonaslagoni** [comment]
@all-contributors please add @ritik307 for doc

**Pull request merged**